api/python/cellxgene_census/src/cellxgene_census/_util.py

```diff
@@ -18,3 +19,7 @@ def _uri_join(base: str, url: str) -> str:
         p_url.fragment,
     ]
     return urllib.parse.urlunparse(parts)
+
+def _extract_census_version(census: soma.Collection):
```

Comment:
I created a live corpus unit test for this method. This should ensure that this parsing method remains consistent across releases.

Reply:
this code has some lint - please run (an up to date) pre-commit across it